public/app/features/data-connections/tabs/DataSources/DataSourcesNew.tsx

```diff
@@ -0,0 +1,7 @@
+import React, { ReactElement } from 'react';
+
+import { NewDataSource } from 'app/features/datasources/components/NewDataSource';
+
+export function DataSourcesNew(): ReactElement | null {
+  return <NewDataSource />;
```

Comment:
Is this abstraction to keep things coming from a single place for data-connections? I'm not sure what the plans are for this feature but right now this component feels a bit overkill.

Reply:
That's a nice catch mate. Yeah, I was planning to have some extra logic in these "Tab components", but they ended up being quite simple and unnecessary. I think I mostly just forgot to clean them up 🙏

Reply:
Fixed in https://github.com/grafana/grafana/pull/52436/commits/32293c591f74bbb414c6a8f9a565dd358aa03a2b.